hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestMemStoreChunkPool.java

```diff
@@ -240,19 +243,24 @@ public void testPutbackChunksMultiThreaded() throws Exception {
     ChunkCreator.instance = newCreator;// Replace the global ref with the new one we created.
                                              // Used it for the testing. Later in finally we put
                                              // back the original
+    final Throwable[] exceptions = new Throwable[1];
     final KeyValue kv = new KeyValue(Bytes.toBytes("r"), Bytes.toBytes("f"), Bytes.toBytes("q"),
         new byte[valSize]);
     try {
       Runnable r = new Runnable() {
         @Override
         public void run() {
-          MemStoreLAB memStoreLAB = new MemStoreLABImpl(conf);
-          for (int i = 0; i < maxCount; i++) {
-            memStoreLAB.copyCellInto(kv);// Try allocate size = chunkSize. Means every
-                                         // allocate call will result in a new chunk
+          try {
+            MemStoreLAB memStoreLAB = new MemStoreLABImpl(conf);
+            for (int i = 0; i < maxCount; i++) {
+              memStoreLAB.copyCellInto(kv);// Try allocate size = chunkSize. Means every
```

Comment:
Just put the comment on top of this line?

Reply:
Modified